data/event-schemas/examples/m.call.negotiate.yaml

```diff
@@ -5,8 +5,7 @@
     "version" : "1",
     "party_id": "67890",
     "call_id": "12345",
-    "lifetime": 10000,
```

Comment:
negotiate still has the lifetime field though: https://github.com/matrix-org/matrix-spec-proposals/pull/2746/files#diff-4d65532ebdb47ba1fd5ea93342395abfeaab0e911360c0bd8d8839be90afd7dfR176

Reply:
Right, sorry about that.